tests/Properties/HashMapLazy.hs

```diff
@@ -323,28 +323,28 @@ tests =
         \(Fn2 f) (x :: HMK A) (y :: HMK B) -> isValid (HM.differenceWith f x y)
       ]
     , testGroup "intersection"
-      [ testProperty "model" $
+      [ testProperty "model" $ QC.withMaxSize 200 $
         \(x :: HMKI) (y :: HMKI) ->
           toOrdMap (HM.intersection x y) === M.intersection (toOrdMap x) (toOrdMap y)
-      , testProperty "valid" $
+      , testProperty "valid" $ QC.withMaxSize 200 $
```

Comment:
Why does this test take a long time? I'd much rather have a slightly more complex `valid` than weaken validity tests. Also, should we reuse the maps generated for the previous test? `.&&.` will let us know which part failed.

Reply:
I dug through the validity code, and it all looks like it should be fast. That makes me wonder why this test would take a long time